ida/data.yml

```diff
@@ -318,6 +316,11 @@ functions:
   0x14192E720: CollisionFree
 
 classes:
+  std::vector:
+    funcs:
+      0x1415f1A90: dtor
```

Comment:
[`std::vector<T>::_Tidy()`](https://github.com/microsoft/STL/blob/3eac329d1f614ecf138d96c22a3b02f87076bc4a/stl/inc/vector#L2039) where [`std::is_trivially_destructible_v<T>`](https://en.cppreference.com/w/cpp/types/is_destructible) [is `true`](https://github.com/microsoft/STL/blob/3eac329d1f614ecf138d96c22a3b02f87076bc4a/stl/inc/xmemory#L1110C1-L1129).